src/components/select/Select.tsx

```diff
@@ -1,13 +1,15 @@
-import { Select as AntDSelect, Spin } from 'antd'
+import { Select as AntDSelect } from 'formik-antd'
+import { Spin } from 'antd'
 import { LoadingOutlined } from '@ant-design/icons'
 import type { SelectProps as AntDSelectProps } from 'antd'
 import { CaretDown, MagnifyingGlass, XCircle } from '@phosphor-icons/react'
 import { useState } from 'react'
 
 type SelectProps = Expand<AntDSelectProps> & {
+  name: string
   size?: 'large' | 'middle' | 'small'
   mode?: '' | 'multiple'
-  onChange?: (value: string | string[]) => void
+  onChange?: (value: string) => void
```

Comment:
Interesting, how does the onChange work with multiselect if it's passing through only a string? Also is `onChange` not defined in the `AntDSelectProps`? I can't remember if this is something I added to try to convince storybook prop tables to render this prop, or because the type actually needs to be defined haha (same as the other props here too, actually)